include/geode/mesh/core/regular_grid.h

```diff
@@ -67,6 +67,8 @@ namespace geode
         friend class bitsery::Access;
 
     public:
+        RegularGrid();

```

Comment:
How do you define the creation of the grid with no parameters ?

Reply:
You don't. This is wrong at the moment but it simplifies SO MUCH this PR. 
In v11, we will need this constructor to use the Grid as a Block mesh. Then, we will add more methods to the builder to create the grid later with some parameters.